src/main/java/com/googlecode/mgwt/ui/client/util/impl/FireFoxCssUtilImpl.java

```diff
@@ -100,13 +100,13 @@ public native void setTransFormOrigin(Element el, int x, int y) /*-{
   }-*/;
 
 	@Override
-	public native void setTransistionTimingFunction(Element element, String string) /*-{
+	public native void setTransistionTimingFunction(Element el, String string) /*-{
 		el.mozTransitionTimingFunction = string;
   }-*/;
 
 	@Override
 	public void setTranslateAndZoom(Element el, int x, int y, double scale) {
-		String cssText = null;
+		final String cssText;
```

Comment:
What were your thoughts in making `cssText` `final` rather than initializing it to `null`?